spec/support/api_schema_matcher.rb

```diff
@@ -14,4 +14,31 @@
 
     true
   end
+
+  def build_json_schema_document_store
+    # load all the schema/*json files that do *not* contain an external $ref
```

Comment:
what if we want to include definitions in a file and reference them from other schema partials? in that case, we might schemas with `ref` that we'd want to load.